src/main/java/pageObjects/DevicePage.java

```diff
@@ -132,131 +134,136 @@ public void successfullyCreateDevice() {
         locationsList.click();
         locationsRomaniaListElement.click();
         saveDeviceButton.click();
-        wait.until(ExpectedConditions.visibilityOf(successToaster));
-        wait.until(ExpectedConditions.invisibilityOf(successToaster));
-        wait.until(ExpectedConditions.visibilityOf(filterButton));
-        filterBySerialNo(serialNo + "");
     }
 
-    public void successfullyEditADeviceName() {
-        successfullyCreateDevice();
+    public void openColumns(WebElement element) {
+        columnsButton.click();
+        ((JavascriptExecutor) driver).executeScript("arguments[0].scrollIntoView(true);", element);
+        elementToLoad(element);
+        element.click();
+        WebElement element2 = driver.switchTo().activeElement();
+        element2.sendKeys(Keys.ESCAPE);
+    }
+
+    public void openEdit(){
         wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("device_dropdown_" + deviceName)));
         driver.findElement(By.id("device_dropdown_" + deviceName)).click();
         wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("device_edit_" + deviceName)));
         driver.findElement(By.id("device_edit_" + deviceName)).click();
-        wait.until(ExpectedConditions.urlToBe(Constants.DEVICE_DETAILS));
+    }
+
+    public void openAssign(){
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("device_dropdown_" + deviceName)));
+        driver.findElement(By.id("device_dropdown_" + deviceName)).click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("device_assign_" + deviceName)));
+        driver.findElement(By.id("device_assign_" + deviceName)).click();
+    }
```

Comment:
is there any difference between these methods? you can make a single method that takes different parameters for the Ids